.github/workflows/create-v1.0-pull-request.yml

```diff
@@ -45,6 +45,30 @@ jobs:
       run: |
         curl -fsSL https://github.com/github/hub/raw/master/script/get | bash -s 2.14.1
         bin/hub pull-request -b "$BASE" -h "$GITHUB_REF" -m "$MESSAGE_TITLE" -m "$MESSAGE_BODY" -r "$REVIEWERS" -a "$ASSIGNEDTO" -l "$LABELS"
+    
+  queue-workflows:
+    needs: create-pull-request
+    runs-on: ubuntu-latest
+    steps: 
+      - name: Queue Api Level Lint
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: "Checks the SDK only using APIs from the targeted API level"
```

Comment:
> Required. The name or ID of the workflow to trigger and run. This is the name declared in the YAML, not the filename
[reference](https://github.com/benc-uk/workflow-dispatch#workflow)

Reply:
That is the current name on the workflow

Reply:
Should I include a name change to this workflow in this PR,  for the sake of being more concise?

Reply:
sorry, I read that sentence the other way around...